src/algorithms/naivefinder.jl

```diff
@@ -61,7 +65,8 @@ function naivefinder(
                 frame = strand == '+' ? framedict[location.start % 3] : framedict[(seqlen - location.stop + 1) % 3]
                 start = strand == '+' ? location.start : seqlen - location.stop + 1
                 stop = start + length(location) - 1
-                push!(orfs, ORF(start:stop, strand, frame, 0.0))
+                id += 1
+                push!(orfs, ORF(get_variable_name(sequence), start:stop, strand, frame, NaiveFinder(), scheme))
```

Comment:
Is there any performance cost to `get_variable_name()`? You're doing it on every iteration, but it looks like it could be pulled outside the loop so you only do it once, right?

Reply:
You are right! Thanks for this suggestion, it reduced the finding time by some microseconds. Updated benches at top.